dev_cycle/terraform/install.sh

```diff
@@ -22,13 +25,20 @@ apt-get -y install docker-ce docker-ce-cli containerd.io qemu
 usermod -a -G docker ubuntu
 
 # install linuxkit
+
 export GOPATH=~/go
 mkdir -p $GOPATH
-go get -u github.com/linuxkit/linuxkit/src/cmd/linuxkit
-cp $GOPATH/bin/linuxkit /usr/local/bin/
+go get github.com/linuxkit/linuxkit/src/cmd/linuxkit
```

Comment:
using a different approach here since we cannot choose any version through `go get -u`